web/src/components/EventListItem.tsx

```diff
@@ -26,40 +26,37 @@ import { EXCERPT_MAX_CHARS } from '../constants'
 import useWindowDimensions from '../hooks/useWindowDimensions'
 import Link from './base/Link'
 
-const StyledListItem = styled(ListItem)(() => ({
-  '& .MuiListItemSecondaryAction-root': {
-    top: 32,
-  },
-}))
+const StyledListItem = styled(ListItem)`
+  [class*='MuiListItemSecondaryAction-root'] {
+    top: 32px;
+  }
+`
```

Comment:
🙃 This is fine like this, but jfyi: You can also use the object syntax without the arrow function, e.g:
```suggestion
const StyledListItem = styled(ListItem)({
  '& .MuiListItemSecondaryAction-root': {
    top: 32,
  },
})
```